content/e/ppw/2023-06-17-ppw-ledwo-legalne-3.md

```diff
@@ -16,6 +16,9 @@ source = "[Official PpW Facebook]"
 - ["[Adam Wong](@/w/adam-wong.md)", "[Goblin](@/w/goblin.md)", {s: "Tables Match"}]
 - ["[Ricardo Diesel](@/w/ricardo-diesel.md)", "[Isnorr](@/w/isnorr.md)"]
 - ["[Mister Z](@/w/mister-z.md)", "[Johnny Blade](@/w/johnny-blade.md)"]
+{s: "Singapore Cane match"}
```

Comment:
If this is correct and there are two matches between Zet and Blade on the card, then this one should be:

```suggestion
- - "[Mister Z](@/w/mister-z.md)"
  - "[Johnny Blade](@/w/johnny-blade.md)"
  - s: "Singapore Cane match"
```